src/hooks/useMediaQuery.ts

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+
+export function useMediaQuery(query: string) {
+  const [value, setValue] = React.useState(false);
+
+  React.useEffect(() => {
+    function onChange(event: MediaQueryListEvent) {
```

Comment:
```suggestion
   const onChange = (event: MediaQueryListEvent) => {
      setValue(event.matches);
    };
```
컴포넌트 내부에서의 함수는 화살표 함수로 선언하기로 했던 것 같습니다!

Reply:
앗 수정하겠습니다